dbt_subprojects/dex/models/_projects/lifi/optimism/lifi_v2_optimism_trades.sql

```diff
@@ -101,23 +96,3 @@ left join {{ source('tokens', 'erc20') }} erc20a
 left join {{ source('tokens', 'erc20') }} erc20b
     on erc20b.contract_address = dexs.token_sold_address
     and erc20b.blockchain = 'optimism'
```

Comment:
```suggestion
left join {{ source('tokens', 'erc20') }} erc20b
    on erc20b.contract_address = dexs.token_sold_address
    and erc20b.blockchain = 'optimism'
```

while we're here, should we remove erc20 metadata too, and apply at the end with prices? similar to dex?

Reply:
I drafted a similar enrichment macro in this PR.